modules/sbom-rhacs.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities/scanner-generate-sbom.adoc
+:_mod-docs-content-type: CONCEPT
+[id="sbom-rhacs_{context}"]
+= Red Hat Advanced Cluster Security and SBOMs
+
+SBOMs can contain different types of information, depending on the methods of data collection and how they are generated. Cybersecurity & Infrastructure Security Agency (CISA) provides a document, link:https://www.cisa.gov/resources-tools/resources/types-software-bill-materials-sbom[Types of Software Bill of Materials (SBOM)], that summarizes the types of SBOMs.
+
+The type of SBOM that {product-title-short} generates is "Analyzed," which is defined by CISA as an "SBOM generated through analysis of artifacts (e.g., executables, packages, containers, and virtual machine images) after its build." The SBOM generated by {product-title-short} is in link:https://spdx.org/rdf/terms/[System Package Data Exchange (SPDX) 2.3] format.
```

Comment:
Consider listing the benefits of an analyzed type SBOM from [Types of Software Bill of Material (SBOM) Documents](https://www.cisa.gov/sites/default/files/2023-04/sbom-types-document-508c.pdf)

Such as:

- Provides visibility without an active development environment.
- Does not need access to the build process.
- May find hidden dependencies missed by other SBOM type creation tools.